PortalHindcasts.R

```diff
@@ -1,19 +1,13 @@
-library(dplyr)
-library(lubridate)
-library(readr)
-library(portalr)
 source('forecast_tools.R')
-source('models/model_functions.R')
-
+source('tools/model_functions.R')
 
```

Comment:
this looks like it will work for the hindcasting. I'll take a closer look when the primary forecasting stuff is up and running.